client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -30,9 +35,69 @@ class ThemeEnhancements extends Component {
 		handleAutosavingToggle: PropTypes.func.isRequired,
 		isSavingSettings: PropTypes.bool,
 		isRequestingSettings: PropTypes.bool,
-		fields: PropTypes.object,
+		fields: PropTypes.object
 	};
 
+	constructor( props ) {
+		super( props );
+		this.getInfiniteMode = this.getInfiniteMode.bind( this );
```

Comment:
These bindings in the constructor can be dropped by defining the methods that need to be bound to the component's scope with the following pattern :

```
updateInfiniteMode = ( radio ) => {
...
};

By using an arrow function you lock the scope of the function to the class that is being definied.